src/library/Uno.Material/Styles/Controls/BottomNavigationBarItem.xaml

```diff
@@ -0,0 +1,152 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:controls="using:Uno.Material.Controls"
+					xmlns:converters="using:Uno.Material.Converters">
+
+	<StaticResource x:Key="BottomNavLabelFocusColorBrush"
+					ResourceKey="OnPrimaryBrush" />
+	<x:Double x:Key="BottomNavLabelFontSize">12</x:Double>
+	<StaticResource x:Key="BottomNavIconFocusColorBrush"
+					ResourceKey="OnPrimaryBrush" />
+	<StaticResource x:Key="BottomNavLabelColorBrush"
+					ResourceKey="OnPrimaryMediumBrush" />
+	<StaticResource x:Key="BottomNavIconColorBrush"
+					ResourceKey="OnPrimaryMediumBrush" />
+	<StaticResource x:Key="BottomNavNotifBackgroundColorBrush"
+					ResourceKey="ErrorBrush" />
+	<StaticResource x:Key="BottomNavNotifLabelColorBrush"
+					ResourceKey="OnErrorBrush" />
+
+	<converters:FromEmptyStringToValueConverter x:Name="EmptyToCollapsed"
```

Comment:
@MatFillion and @Massimo37 Do we want to keep them here ?

Reply:
@MatFillion do you want me to add a Converters.xaml or push the PR as is and do it seperately ?

Reply:
We can move them but what we really need to do is make them with specific names, we don't want to conflict with names applications could use as global resources. Then our code would use their converters unintentionally

Reply:
So we can prepend our converters with something like UnoMaterial__NAME...

Reply:
I see you comment below, I will rename converters to use "Material_NAME"